docs/pages/guides/email-and-password/password-reset.md

```diff
@@ -54,7 +55,7 @@ app.post("/reset-password", async () => {
 		});
 	}
 
-	const verificationToken = await createPasswordResetToken(userId);
+	const verificationToken = encodeHex(await createPasswordResetToken(userId));
```

Comment:
Why is this required?

Reply:
I assumed you were meant to do this since there is an import for it.
`import { encodeHex } from "oslo/encoding";`